docs/getting-started/README.md

```diff
@@ -1,6 +1,7 @@
-# Before you begin
+# Getting Started with a Knative Sandbox
 
-Before you can get started with Knative you must set up a development environment.
+## Before you begin
+Before you can get started with a Knative Sandbox deployment you must set up a development environment.
```

Comment:
Is the use of "sandbox" not going to be confusing here because of https://github.com/knative-sandbox?

Reply:
FFS you're totally right.  

knative-sandbox is a terrible name for what knative-sandbox is

Reply:
given I don't want to open THAT can of worms though, any thoughts on a different name here? 

The user said that going to getting started and seeing a local deploy made him feel like Knative was a local tool which is not the case. 

I want to make it clear to users "This is a thing to play with, not put in production, but if you wanted to put Knative in production you could"